test_runner/src/main/kotlin/ftl/gc/GcAndroidTestMatrix.kt

```diff
@@ -57,6 +58,7 @@ object GcAndroidTestMatrix {
             .setAdditionalApks(additionalApkGcsPaths.mapGcsPathsToApks())
             .setFilesToPush(otherFiles.mapToDeviceFiles())
             .setEnvironmentVariables(args, androidTestConfig)
+            .setDontAutograntPermissions(!args.isShouldAutoGrantPermissions)
```

Comment:
Changing the name to something similar to gcloud's would allow us to get rid of `!`